ctapipe/image/muon/fitting.py

```diff
@@ -83,6 +87,23 @@ def _psf_neg_log_likelihood(params, x, y, weights):
         (np.log(sigma) + 0.5 * ((pixel_distance - radius) / sigma)**2) * weights
     )
 
+def all_to_value(*args, unit):
```

Comment:
Nice, that's a really general function. Move it somewhere more appropriate?

Reply:
yes ... do you have an idea where?

Reply:
I assume moving functions to better places can easily done in the future. This PR should maybe not depend on it.

Reply:
moved it to utils